### ISSUE TITLE
Not Compatible With Flatpak Steam

### ISSUE THREAD

**Issue by cprin21** [ralgar/proton-mo2-installer#9]
Related to New Vegas and trying to get that to work, the install script seems to want to apply the custom proton version to the native Steam application instead of Flatpak Steam if Flatpak Steam is installed and the game is installed on/via Flatpak Steam.

Being able to use this to install MO2 for New Vegas on Flatpak Steam would let me test to see if it works there.

**Comment by ralgar** [comment]
Yeah, Flatpak is not currently compatible. I *was* in the middle of working on it, but for some reason I couldn't get Flatpak Steam to run any games on my laptop, so I put Flatpak support on the backburner.

It is still on my radar though, eventually, but at the moment I'm working on other things that I need to finish up first. Keep an eye out for a future update, if you're still interested!
